warehouse/utils/utils.go

```diff
@@ -1062,6 +1062,14 @@ type FilterBy struct {
 	Value interface{}
 }
 
+type ColumnInfoT struct {
+	Name  string
+	Value interface{}
+	Type  string
+}
+
+type ColumnsInfo []ColumnInfoT
```

Comment:
* let's avoid the T suffix
* In go it is not very idiomatic to define a type for slices of other types. We can simply use `[]ColumnInfo`, which is more readable as well, because it is clear it is a slice, and you now that is a slice of `ColumnInfo` without further inspection. Further more it only cost an extra character if you think about it `s` vs `[]`


```suggestion
type ColumnInfo struct {
	Name  string
	Value interface{}
	Type  string
}
```

Reply:
Done.